pyg_lib/csrc/sampler/cpu/neighbor_kernel.cpp

```diff
@@ -135,25 +135,50 @@ class NeighborSampler {
 
     // Case 2: Sample with replacement:
     else if (replace) {
-      for (size_t i = 0; i < count; ++i) {
-        const auto edge_id = generator(row_start, row_end);
-        add(edge_id, global_src_node, local_src_node, dst_mapper,
-            out_global_dst_nodes);
+      if (row_end < (1 << 16)) {
+        int arr[count];
```

Comment:
`count` is not a constant expression, this will probably fail with some compilers.

Reply:
Fixed.